assets/assets.json

```diff
@@ -452,8 +452,8 @@
 		"off": true,
 		"title": "LTU: EasyList Lithuania",
 		"lang": "lt",
-		"contentURL": "https://margevicius.lt/easylistlithuania.txt",
-		"supportURL": "https://margevicius.lt/2014/06/26/easylist_lithuania/"
+		"contentURL": "https://raw.githubusercontent.com/EasyList-Lithuania/easylist_lithuania/master/easylistlithuania.txt",
```

Comment:
Do we need to update our version of this list / link?

Reply:
We're using `url: String::from("https://raw.githubusercontent.com/EasyList-Lithuania/easylist_lithuania/master/easylistlithuania.txt"),` from https://github.com/brave/adblock-rust/blob/master/src/filter_lists/regions.rs#L247

So it seems fine, we already updated

Reply:
terrific, thanks for checking